src/plugins/discover/public/application/_discover.scss

```diff
@@ -104,7 +104,7 @@ discover-app {
   top: $euiSizeXS;
 }
 
-[fixed-scroll] {
+.fixed-scroll {
```

Comment:
Since we're changing this, can we make it use the `.dsc` prefix?

Reply:
sure, thx & done!